docs/getting-started/dockstore-workflows.rst

```diff
@@ -114,31 +114,25 @@ This is the newest way of getting content onto Dockstore and is by far the most
 GitHub Apps, Dockstore can react to changes on GitHub as they are made, keeping Dockstore synced
 with GitHub automatically.
 
-To install our GitHub App on either a repository or organization,
-navigate to the ``/my-workflows`` page and click add workflow. Follow the steps for GitHub Apps and
-you will be redirected to GitHub where you can select which repositories to install the Dockstore
-GitHub app on. The process is shown in the following images.
+Installing the GitHub App is simply -- navigate to ``/my-workflows``, click the ``+`` button on the left hand sidebar, select ``Register using GitHub Apps``, and then click
+``+ Manage Dockstore Installation on GitHub``. You'll then be redirected to GitHub where you can select which repositories can be accessed by the GitHub app.
 
-.. figure:: /assets/images/docs/gh-app-reg-1.png
-   :alt: GitHub App Registration on Dockstore
+.. image:: /assets/images/docs/add-workflow-button.png
+   :width: 40 %
 
-   Select Register Using GitHub Apps to get a link to our installation page
+.. image:: /assets/images/docs/register-workflow-github-apps.png
+   :width: 40 %
 
-.. figure:: /assets/images/docs/gh-app-reg-2.png
-   :alt: GitHub App Organizations Page
-
-   Select an organization
-
-.. figure:: /assets/images/docs/gh-app-reg-3.png
-   :alt: GitHub App Organization Install Page
+.. image:: /assets/images/docs/gh-app-reg-1.png
+   :width: 40 %
 
    Install our GitHub App on either all repositories in an organization or on specific repositories
 
 Once you've installed our GitHub app on a repository or organization, you'll need to add a ``/.dockstore.yml`` file to
 the root directory of a branch of the repository that contains your workflow. This file contains information like
 workflow path, test parameter file, workflow name, etc. When a push is made on GitHub to a branch
 with a ``/.dockstore.yml``, Dockstore will add that branch to the corresponding workflow on Dockstore. If the
-workflow doesn't already exist on Dockstore, one will be created.
+workflow doesn't already exist on Dockstore, one will be created. Note that if you have more than workflow in your repo, you can describe multiple in one ``/.dockstore.yml`` file to create multiple workflow entries.
```

Comment:
typo/suggestion: more than **one** workflow, you can describe each one in a single ``/.dockstore.yml``

Reply:
Good spot, and a good opportunity to make it even more explicit. How about this?
`Note that a single ``/.dockstore.yml`` file can describe multiple workflows, if all of those workflows are in the same repository. This will create multiple workflow entries on Dockstore from one ``/.dockstore.yml`` file, one entry for each workflow.`